app/backend/src/couchers/models.py

```diff
@@ -76,50 +95,69 @@ class User(Base):
     geom_radius = Column(Float, nullable=True)
     # the display address (text) shown on their profile
     city = Column(String, nullable=False)
+    hometown = Column(String, nullable=True)
 
     joined = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
     last_active = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
     # display name
     name = Column(String, nullable=False)
     gender = Column(String, nullable=False)
+    pronouns = Column(String, nullable=True)
     birthdate = Column(Date, nullable=False)  # in the timezone of birthplace
 
     # name as on official docs for verification, etc. not needed until verification
     full_name = Column(String, nullable=True)
 
     hosting_status = Column(Enum(HostingStatus), nullable=True)
+    meetup_status = Column(Enum(MeetupStatus), nullable=True)
 
     # verification score
     verification = Column(Float, nullable=True)
     # community standing score
     community_standing = Column(Float, nullable=True)
 
     occupation = Column(String, nullable=True)
-    about_me = Column(String, nullable=True)
+    education = Column(String, nullable=True)
+    about_me = Column(String, nullable=True, default="Current Mission:\n\nWhat I do in my free time:\n")
+    my_travels = Column(String, nullable=True, default="Why I use Couchers:\n\nMy favorite travel story:\n")
+    things_i_like = Column(String, nullable=True, default="Art:\n\nBooks:\n\nMovies:\n\nMusic:\n")
```

Comment:
These defaults should be defined in the frontend, and models should have no defaults (i.e., be null)

Reply:
@lucaslcode Do you agree about the defaults? It was your request for defaults in the first place

Reply:
Yes that's fine (sorry)

Reply:
Not a problem! Just looking for consensus!